designs/2019-pass-cwd-from-cli-engine/README.md

```diff
@@ -0,0 +1,50 @@
+- Start Date: 2019-07-25
+- RFC PR: https://github.com/eslint/eslint/pull/12021
+- Authors: Eric Wang
+
+# Pass cwd from cli engine
+
+## Summary
+Make the CLIEngine `cwd` option available to rules to avoid the misalignment between it and process.cwd()
+
+see: https://github.com/eslint/eslint/issues/11218
+
+## Motivation
+To avoid the misalignment between it and process.cwd().
```

Comment:
I have a few questions. Do what kinds of rules need `cwd`? Why is not `context.getFilename()` enough?

Reply:
Because we have quite a few custom rules that need to get the relative path of the filename to the cwd.
Should I add that to the README as well?

Reply:
Oh, I got it. Yes, please.